Booting/linux-bootstrap-5.md

```diff
@@ -164,7 +164,7 @@ After it we get `.text` section address and jump to it:
 Last preparation before kernel decompression
 --------------------------------------------------------------------------------
 
-`.text` sections starts with the `relocated` label. For the start there is clearing of the `bss` section with:
+`.text` sections start with the `relocated` label. For the start there is clearing of the `bss` section with:
```

Comment:
Hi @0xAX, the changes have been made. I also added 'The' at the beginning
to make it grammatically correct. Thanks for checking for correctness for
me!

On Thu, Oct 15, 2015 at 1:06 PM 0xAX <notifications@github.com> wrote:

> In Booting/linux-bootstrap-5.md
> <https://github.com/0xAX/linux-insides/pull/250#discussion_r42151296>:
>
> > @@ -164,7 +164,7 @@ After it we get `.text` section address and jump to it:
> >  Last preparation before kernel decompression
> >  --------------------------------------------------------------------------------
> >
> > -`.text` sections starts with the `relocated` label. For the start there is clearing of the `bss` section with:
> > +`.text` sections start with the `relocated` label. For the start there is clearing of the `bss` section with:
>
> Hi @ruthgrace <https://github.com/ruthgrace>,
>
> There must be: '.text section' instead of '.text sections', can you please
> fix this and put back starts
>
> Thank you
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/0xAX/linux-insides/pull/250/files#r42151296>.
>